### PR TITLE
feat(ratelimit): Added 429 error handling

### PR DESCRIPTION
**Pull request by shrav**

### CODE REVIEW

File: src/Lob/Resource.php [hunk 1]
```diff
@@ -121,6 +121,13 @@ protected function sendRequest($method, $version, $clientVersion, $path,
 
             if ($statusCode === 422)
                 throw new ValidationException($errorMessage, 422);
+
+            // @codeCoverageIgnoreStart
+            // must induce serverside error to test this, so not testable
+            if ($statusCode === 429)
+                throw new RateLimitException($errorMessage, 429);
```
Comment (dmlittle) [inline src/Lob/Resource.php]: You forgot to import the `RateLimitException` at the top after all imports

```php
use Lob\Exception\RateLimitException;
```

Comment (shrav) [inline src/Lob/Resource.php]: fixed.

**Comment by robinjoseph08** [comment]
LGTM

**Comment by elnaz** [comment]
LGTM

**Pull request merged**